OracleTuxedo/OracleTSAM/README.md

```diff
@@ -0,0 +1,176 @@
+Oracle TSAM Plus on Docker
+===============
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. This image has also been certified with [Oracle Container Cloud Service (OCCS)](https://cloud.oracle.com/container). For more information about **Oracle Tuxedo System and Applications Monitor Plus (TSAM Plus)** please see the [Oracle TSAM Plus Online Documentation](http://docs.oracle.com/cd/E72452_01/tsam/docs1222/index.html).
+
+This project offers Dockerfile for building:
+ * Oracle TSAM Plus 12c Release 2 (12.2.2) + Rolling Patch 004
+
+## Dependencies
+This project depends on the [Oracle Server JRE 8 Docker Image](../..//OracleJava). So, before you proceed to build this image, make sure the image `oracle/serverjre:8` has been built locally or is accessible in a remote Docker registry.
+
+**IMPORTANT:** You will have to provide the required installation binaries and put them into the `dockerfiles/<version>` folder.
+
+For this image following installation media / binaries are required:
+
+* [Oracle TSAM Plus 12.2.2 GA Installer](http://www.oracle.com/technetwork/middleware/tuxedo/downloads/index.html) - `tsam122200_64_Linux_x86.zip`
+* [Oracle TSAM Plus 12.2.2 Rolling Patch 004](https://updates.oracle.com/Orion/Services/download/p25530287_12220_Linux-x86-64.zip?aru=21140450&patch_file=p25530287_12220_Linux-x86-64.zip) - `p25530287_12220_Linux-x86-64.zip`
```

Comment:
However, the reason why we were recommending RP003 is, in that patch we introduced a brand new TSAM Dashboard, which is powered by Oracle JET. There were some other important enhancements in that patch. So, basically TSAM GA + RP003 makes TSAM a new minor release, internally we call it 12.2.2.1. So, we definitely want to show these latest features.
@Djelibeybi Can I remove the RP as a requirement but still leave it as an option for the users who have rights to obtain the patch?